performance_regression_gradual_grow_throughput.py

```diff
@@ -54,6 +55,15 @@ def throttle_steps(self, workload_type):
                                severity=Severity.CRITICAL).publish()
         return throttle_steps[workload_type]
 
+    def step_duration(self, workload_type):
+        step_duration = self.params["perf_gradual_step_duration"]
+        if workload_type not in step_duration:
+            TestFrameworkEvent(source=self.__class__.__name__,
```

Comment:
Shouldn't we be ok if the duration is built into a cmd command?

Reply:
Yes, but we expect that step_duration should be explicitly defined for every workload_type If the duration is built in, it should be set to None

Reply:
I think this UX diff isn't that big, we can put this in
and in a followup, see how to replace it with same functionality we use in longivties